.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Build and push multi-platform docker images
+
+on:
+  push:
+    tags:
+      - '^v[0-9]{2}\.[0-9]{2}(\.[0-9]{1,2})?$'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version'
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Setup Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    - name: Cache Docker layers
+      uses: actions/cache@v2
+      with:
+        path: /tmp/.buildx-cache
+        key: ${{ runner.os }}-buildx-${{ github.sha }}
+        restore-keys: |
+          ${{ runner.os }}-buildx-
+
+    - name: Build and push Docker image
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        push: true
+        platforms: linux/amd64,linux/arm64,linux/arm/v7
+        tags: |
+          shahanafarooqui/lightningd:${{ github.event.inputs.version }}
+          shahanafarooqui/lightningd:latest
```

Comment:
The variables are now configured on GH Actions for the ElementsProject/lightning repo, so we can change these to point to `elementsproject/lightningd` now :-)

Reply:
Oh, I accidentally pushed with my test namespace :woman_facepalming: . Thanks, fixed.